### PR TITLE
Update for (src/Forms/ImportImages.cs)

### PR DESCRIPTION
**Pull request by ivandrofly**
Status:
- [x] - Double check
- [x] - Ready

### CODE REVIEW

File: src/Forms/ImportImages.cs [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 using System.Windows.Forms;
+using System.Collections;
 
 namespace Nikse.SubtitleEdit.Forms
 {
     public sealed partial class ImportImages : PositionAndSizeForm
     {
         public Subtitle Subtitle { get; private set; }
-
+        private readonly Hashtable FilesAlreadyInList;
```
Comment (xylographe) [inline src/Forms/ImportImages.cs]: Perhaps, `private readonly HashSet<string> ListedFileNames = new HashSet<string>();` ?

Comment (ivandrofly) [inline src/Forms/ImportImages.cs]: > Perhaps, private readonly HashSet<string> ListedFileNames = new HashSet<string>(); ?

yeap

File: src/Forms/ImportImages.cs [hunk 2]
```diff
@@ -132,5 +138,43 @@ private void ImportImages_KeyDown(object sender, KeyEventArgs e)
             }
         }
 
+        private void contextMenuStrip1_Opening(object sender, System.ComponentModel.CancelEventArgs e)
+        {
+            if (listViewInputFiles.Items.Count < 0)
+                return;
```
Comment (niksedk) [inline src/Forms/ImportImages.cs]: ´ e.Cancel = true;´   ?

**Comment by ivandrofly** [comment]
Done!

**Pull request merged**